docs/modules/clients/pages/java-thin-client.adoc

```diff
@@ -0,0 +1,1126 @@
+= {client-name}
+:page-api-reference: https://docs.hazelcast.org/docs/{page-latest-supported-java-client}/javadoc
```

Comment:
This will link to the corresponding Javadocs for the _non-thin_ client. Not sure how we generate separate Javadocs for _thin client_ but we will need to remove the link or change it i.e. expand the variables to include thin and fat client options to link to.

Reply:
Let's remove for now